pkg/crc/machine/machine.go

```diff
@@ -246,6 +246,14 @@ func Start(startConfig StartConfig) (StartResult, error) {
 	}
 	logging.Info("CodeReady Containers VM is running")
 
+	// Post VM start immediately update SSH key and copy kubeconfig to instance
+	// dir and VM
+	if !exists {
+		if err := updateSSHKeyAndCopyKubeconfigToVM(sshRunner, startConfig, crcBundleMetadata); err != nil {
```

Comment:
It's not just `ToVM`, it's both to VM and to instance dir. Fwiw, I don't really think updating the ssh key and copying the kubeconfig file to the right places need to be in the same helper.